x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_event_log_data_grid.tsx

```diff
@@ -277,6 +291,7 @@ export const RuleEventLogDataGrid = (props: RuleEventLogDataGrid) => {
         totalItemCount={pagination.totalItemCount}
       />
       <EuiDataGrid
+        key={key}
```

Comment:
is that a risky move? should we reset this key when we have new status filters or a new time range?

Reply:
my assumption is wrong here, I tested it locally and everything is working as expected.
@JiaweiWu follow this advice from [here](https://elastic.github.io/eui/#/tabular-content/data-grid#virtualization)